pkg/psmdb/statefulset.go

```diff
@@ -584,7 +584,11 @@ func isEncryptionEnabled(cr *api.PerconaServerMongoDB, replset *api.ReplsetSpec)
 	if err != nil {
 		return false, errors.Wrap(err, "failed to parse replset configuration")
 	}
+
```

Comment:
Since the cr is not used by this function at all, maybe we can remove it from the function signature.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1912/commits/23d9ce0543250918a883de73d66e335ab71ef2ee